android/src/main/java/com/stripeterminalreactnative/Mappers.kt

```diff
@@ -543,28 +559,30 @@ internal fun mapFromPaymentMethodDetailsType(type: PaymentMethodType?): String {
     }
 }
 
-private fun mapFromCardPresentDetails(cardPresentDetails: CardPresentDetails?): ReadableMap =
-    nativeMapOf {
-        putString("brand", cardPresentDetails?.brand)
-        putString("cardholderName", cardPresentDetails?.cardholderName)
-        putString("country", cardPresentDetails?.country)
-        putString("emvAuthData", cardPresentDetails?.emvAuthData)
-        putIntOrNull(this, "expMonth", cardPresentDetails?.expMonth)
-        putIntOrNull(this, "expYear", cardPresentDetails?.expYear)
-        putString("funding", cardPresentDetails?.funding)
-        putString("generatedCard", cardPresentDetails?.generatedCard)
-        putString("last4", cardPresentDetails?.last4)
-        putString("readMethod", cardPresentDetails?.readMethod)
-        putMap("receiptDetails", mapFromReceiptDetails(cardPresentDetails?.receiptDetails))
-        putString("issuer", cardPresentDetails?.issuer)
-        putString("iin", cardPresentDetails?.iin)
-        putString("network", cardPresentDetails?.network)
-        putString("description", cardPresentDetails?.description)
-        putMap("wallet", mapFromWallet(cardPresentDetails?.wallet))
-        putArray(
-            "preferredLocales",
-            convertListToReadableArray(cardPresentDetails?.preferredLocales)
-        )
+private fun mapFromCardPresentDetails(cardPresentDetails: CardPresentDetails?): ReadableMap? =
+    cardPresentDetails?.let {
```

Comment:
previously we were returning the map with all fields set to `null`, want to return just null instead